src/main/java/io/logz/guice/jersey/JerseyServer.java

```diff
@@ -66,8 +66,8 @@ private void configureServer() {
 
         WebAppContext webAppContext = new WebAppContext();
         webAppContext.setServer(server);
-
-        webAppContext.addFilter(GuiceFilter.class, "/*", EnumSet.allOf(DispatcherType.class));
+        if (jettyServerConfiguration != null)
```

Comment:
Put this *before* `webAppContext.setServer(server);` you they won't run you down